targets/targets.json

```diff
@@ -13442,44 +13357,37 @@
             "deep-sleep-latency": 0
         }
     },
-    "TMPM3HQ": {
+    "RDA5981X": {
         "inherits": [
             "Target"
         ],
-        "core": "Cortex-M3",
-        "is_disk_virtual": true,
+        "core": "Cortex-M4F",
+        "public": true,
         "extra_labels": [
-            "TOSHIBA"
-        ],
-        "macros": [
-            "__TMPM3HQ__"
+            "RDA",
+            "UNO_91H",
+            "FLASH_CMSIS_ALGO",
+            "RDA_EMAC"
         ],
         "supported_toolchains": [
-            "GCC_ARM",
             "ARM",
+            "GCC_ARM",
             "IAR"
         ],
+        "macros": [
+            "CMSIS_NVIC_VIRTUAL",
+            "CMSIS_NVIC_VIRTUAL_HEADER_FILE=\"RDA5981_nvic_virtual.h\""
+        ],
         "device_has": [
             "USTICKER",
-            "ANALOGIN",
-            "ANALOGOUT",
-            "CRC",
-            "INTERRUPTIN",
             "PORTIN",
-            "PORTINOUT",
             "PORTOUT",
-            "PWMOUT",
-            "RESET_REASON",
-            "RTC",
+            "PORTINOUT",
+            "INTERRUPTIN",
+            "EMAC",
             "SERIAL",
             "SERIAL_FC",
-            "SLEEP",
-            "SPI",
-            "SPI_ASYNCH",
-            "SPISLAVE",
-            "I2C",
-            "I2C_ASYNCH",
-            "I2CSLAVE",
+>>>>>>> 8366de4000... Removed Toshiba target AdBun-M066, M3H6 and M3HQ.
```

Comment:
this was left from the last rebase , should not be in

Reply:
Updated